### PR TITLE
Dynamic width, peer deps in docs, error state fix

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #35, #38

## Summary:

Fix error state help text so that it clears after selecting a date. 

Add peer dependency information to docs.

Update to latest version of auro-dropdown to use improved template layout so that the input in the trigger is full width.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: scripts/generateDocs.js [hunk 1]
```diff
@@ -199,28 +199,30 @@ function processApiExamples() {
...
       processReadme();
-    });
+  //   });
 
-  }).on('error', (err) => {
-    console.log(chalk.red('ERROR: Unable to fetch README.md file from server.', err));
-  });
+  // }).on('error', (err) => {
+  //   console.log(chalk.red('ERROR: Unable to fetch README.md file from server.', err));
+  // });
```
Comment (blackfalcon) [inline scripts/generateDocs.js:225]: Looks like you are bypassing the original function for a local update. Please delete this code.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by blackfalcon** [comment]
Once commented code is removed, feature update is approved to merge and ship.

File: scripts/generateDocs.js (hunk 1 above)
Comment (jason-capsule42) [reply scripts/generateDocs.js:225]: @blackfalcon I don't want to remove this code. Leaving it in place is a reminder that this code is currently running in a non-standard format that we don't want to support longer than necessary. 

This is a temporary change until we get the template in the generator repo updated per this ticket: https://github.com/AlaskaAirlines/WC-Generator/issues/352.

I'd like to actually prioritize getting ticket 352 completed.

Comment (blackfalcon) [reply scripts/generateDocs.js:225]: We have version control, there is no need to keep commented code around with an ambiguous intent to bring it back. The preferred method is to remove this code with a commit that clearly states that this is what happened.

**Pull request merged**